rpc/src/v1/traits/traces.rs

```diff
@@ -43,6 +46,7 @@ pub trait Traces: Sized + Send + Sync + 'static {
 		delegate.add_method("trace_transaction", Traces::transaction_traces);
 		delegate.add_method("trace_block", Traces::block_traces);
 		delegate.add_method("trace_call", Traces::call);
+		delegate.add_method("trace_sendRawTransaction", Traces::send_raw_transaction);
```

Comment:
`trace_transaction` and `trace_rawTransaction` would have been nice but `trace_transaction` is already used (and irritatingly, not even for a function of the corresponding name: `Traces::transaction_traces`)